friendshipper/src-tauri/src/repo/operations/revert.rs

```diff
@@ -62,13 +63,20 @@ where
             }
         }
 
-        for chunk in self.files.chunks(50) {
-            let mut args: Vec<&str> = vec!["checkout", &branch, "--"];
-            for file in chunk {
-                args.push(file);
-            }
-            self.git_client.run(&args, git::Opts::default()).await?;
+        let mut temp_file = NamedTempFile::new()?;
```

Comment:
Might as well delete the `revert_with_listfile` logic.

Reply:
yup you're right. pushed!